### PR TITLE
Added zone.ee and zone.fi favicon and signing policies

### PR DESCRIPTION
**Pull request by TaaviE**
This PR adds zone.ee and zone.fi to the favicon and signing policies.

The SVG matches the one in zone.ee's BIMI/VMC certificate.

### CODE REVIEW

File: data/favicon.json [hunk 1]
```diff
@@ -339,5 +339,7 @@
 	"yelp.com": "yelp.com.ico",
 	"yoast.com": "yoast.com.png",
 	"youtube.com": "youtube.com.png",
-	"zeit.de": "zeit.de.svg"
+	"zeit.de": "zeit.de.svg",
+	"zone.ee": "zone.ee.svg",
+	"zone.fi": "zone.ee.svg"
```
Comment (lieser) [inline data/favicon.json:344]: Did you have a look how that would look with the icon you include here?

Note that the favicon will only be shown as a small quadratic icon. So the same icon that is used for the website (https://www.zone.ee/static/cropped-favicon-512x512-1-32x32.png) is probably a better fit than the currently included rectangular icon.

**Review by lieser — CHANGES_REQUESTED** [review]
Thanks for adding the domains, but I think a different icon would be more appropriate.

File: data/favicon.json (hunk 1 above)
Comment (TaaviE) [reply data/favicon.json:344]: The choice was between following BIMI/VMC or increased legibility. I'll choose the latter knowing it is fine to do so.

**Comment by TaaviE** [comment]
I've replaced the icon. If possible, a squash merge would look nicer in the commit history.

**Review by lieser — APPROVED** [review]

File: data/favicon.json (hunk 1 above)
Comment (lieser) [reply data/favicon.json]: As a site note, BIMI recommends the usage of an icon with a square aspect ratio https://bimigroup.org/creating-bimi-svg-logo-files/.

Comment (TaaviE) [reply data/favicon.json]: @lieser VMC on the other hand requires a valid trademark. Otherwise yes, we'd use a more square icon.

**Pull request merged**